orchagent/routeorch.cpp

```diff
@@ -1652,6 +1693,7 @@ bool RouteOrch::addRoute(RouteBulkContext& ctx, const NextHopGroupKey &nextHops)
             gRouteBulker.set_entry_attribute(&object_statuses.back(), &route_entry, &route_attr);
         }
     }
+
```

Comment:
Extra line added

Reply:
It's just an empty line between the body of the function and the return value. Pretty much all of the other methods do the same thing.

Reply:
Done